.github/DEVELOPER.md

```diff
@@ -8,15 +8,28 @@
 * kubebuilder
 
 ### Quickstart
-```
-$ git clone https://github.com/keikoproj/iam-manager
-$ cd iam-manager
-$ go mod
-$ go vendor
-```
 
+First, go and fork the Github repo to your own personal project. Once that's
+done, set up a local build environment off of the original Github repo. Then we
+add in your fork'ed repo as a new target for doing git pushes.
+
+    $ go clean -modcache
+    $ go get -v github.com/keikoproj/iam-manager
+    $ cd "$(go env GOPATH)/src/github.com/keikoproj/iam-manager"
+    $ make test
+    $ go mod vendor
+    $ git remote add myfork <your fork>
 
 ### Build project
-```
-$ make
-```
\ No newline at end of file
+
+    $ make
+
+### Running Tests
+
+There are several environment variables that must be set in order for the
+test suite to work.
+
+    $ export KUBECONFIG=$HOME/.kube/config
+    $ export LOCAL=true
```

Comment:
Thanks for cleaning up the documents. I really appreciate that.

This environment variable is defined in Makefile itself. Isn't that not working as expected when you tried in your local?

Reply:
I think the issue I ran into was primarily the `KUBECONFIG` bit.. however, you also need `LOCAL=true` if you're going to do something like run a specific test directly without using the Makefile. I don't mind cleaning up the makefile a little bit (we can make some of these exports a bit more obvious), and continuing to clean up the docs.